lib/plug/ssl.ex

```diff
@@ -298,7 +299,7 @@ defmodule Plug.SSL do
     host = Keyword.get(opts, :host)
     rewrite_on = List.wrap(Keyword.get(opts, :rewrite_on))
     log = Keyword.get(opts, :log, :info)
-    exclude = Keyword.get(opts, :exclude, ["localhost"])
+    exclude = excluded_hosts(opts)
```

Comment:
Unfortunately this is being invoked at compilation time, which is not going to have the behaviour you expect. I think it is best to implement it like this:

    exclude: [binary()] | {mod, f, args}

And the current host will be passed as the first argument to the mfargs. So you will invoke the given mfargs on every unprotected call to see if it should be excluded or not.

Reply:
Thanks, fixed.